codegen-server-test/build.gradle.kts

```diff
@@ -82,7 +82,7 @@ task("generateCargoWorkspace") {
 }
 
 tasks["smithyBuildJar"].dependsOn("generateSmithyBuild")
-
```

Comment:
What was this again for? Will it be faster to do `:assemble` instead of `:build`?

Reply:
yeah `:assemble` will run codegen but no tests, build will also run the tests